IFC4x3/Properties/i/IndicationRodMovement_2EWdqFr25C5QKm2WsoAIDD/Documentation.md

```diff
@@ -0,0 +1 @@
+The movement of the bar which indicates the indication rod position.
```

Comment:
This somehow does not fit the name of the property. Is this correct?

Reply:
The only unresolved issue in this PR. Not clear to me either, needs to check with Domains. Florian has similar problem in another PR.

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/a55801bb3daa1553197b470f6b04e0fcda48397f